pyartcd/pyartcd/pipelines/update_golang.py

```diff
@@ -376,11 +370,11 @@ async def update_golang_streams(self, go_version, builder_nvrs):
             try:
                 build_url = jenkins.get_build_url()
                 body = f"Created by job run {build_url}" if build_url else ""
-                pr = upstream_repo.create_pull(title=title, body=body, base=branch_name, head=f"openshift-bot:{branch_name}")
-                self._logger.info(f"PR created {pr.html_url} for {branch_name} to bump {self.ocp_version} golang builders to {go_version}")
+                pr = upstream_repo.create_pull(title=title, body=body, base=branch, head=f"openshift-bot:{branch_name}")
+                _LOGGER.info(f"PR created {pr.html_url} for {branch_name} to bump {self.ocp_version} golang builders to {go_version}")
                 await self._slack_client.say_in_thread(f"PR created {pr.html_url} for {branch_name} to bump {self.ocp_version} golang builders to {go_version}")
             except GithubException as e:
-                self._logger.warning(f"Failed to create pr to bump golang builder: {e}")
+                _LOGGER.warning(f"Failed to create pr to bump golang builder: {e}")
```

Comment:
I actually like it if the job fails when it fails to create a PR. Could you maybe remove the `try` block?